jenkins/bootstrap.py

```diff
@@ -841,14 +844,29 @@ def setup_root(call, root, repos, ssh, git_cache, clean):
     os.chdir(root_dir)
     logging.info('cd to %s', root_dir)
 
+    # we want to checkout the correct repo for k-s/k but *everything*
+    # under the sun assumes $GOPATH/src/k8s.io/kubernetes so... :(
+    # after this method is called we've already computed the upload paths
+    # etc. so we can just swap it out for the desired path on disk
     with choose_ssh_key(ssh):
         for repo, (branch, pull) in repos.items():
             os.chdir(root_dir)
+            # for k-s/k these are different, for the rest they are the same
+            # TODO(bentheelder,cjwagner,stevekuznetsov): in the integrated
+            # prow checkout support remapping checkouts and kill this monstrosity
+            repo_path = repo
+            if repo == "github.com/kubernetes-security/kubernetes":
+                repo = "k8s.io/kubernetes"
```

Comment:
:man_facepalming: this should be `repo_path` https://github.com/kubernetes/test-infra/pull/6588